src/data/brands.js

```diff
@@ -0,0 +1,56 @@
+export const DEFAULT_BRANDS_RESPONSE_BODY = {
+    "status": "ok",
+    "data": [
+        {
+            "id": 1,
+            "title": "Audi",
+            "logoFilename": "audi.png"
+        },
+        {
+            "id": 2,
+            "title": "BMW",
+            "logoFilename": "bmw.png"
+        },
+        {
+            "id": 3,
+            "title": "Ford",
+            "logoFilename": "ford.png"
+        },
+        {
+            "id": 4,
+            "title": "Porsche",
+            "logoFilename": "porsche.png"
+        },
+        {
+            "id": 5,
+            "title": "Fiat",
+            "logoFilename": "fiat.png"
+        }
+    ]
+};
+export const CUSTOM_BRANDS_RESPONSE_BODY = {
+    "status": "ok",
+    "data": [
+        {
+            "id": 1,
+            "title": "Audi",
+            "logoFilename": "audi.png"
+        },
+        {
+            "id": 2,
+            "title": "BMW",
+            "logoFilename": "bmw.png"
+        },
+        {
+            "id": 3,
+            "title": "Ford",
+            "logoFilename": "ford.png"
+        }
+    ]
+}
```

Comment:
it is a fixture, move it to fixtures folder where you have the test which uses this fixture